Makefile

```diff
@@ -35,7 +35,7 @@ iso_creation: init
 	dd if=$(BUILD_DIR)/$(ISO_NAME).iso bs=2k skip=1 >> ${BUILD_DIR}/tmp.iso
 	mv -f ${BUILD_DIR}/tmp.iso $(BUILD_DIR)/$(ISO_NAME).iso
 
-.PHONY: check
+.PHONY: check_env
```

Comment:
> In makefile .PHONY can be different from the execution.

Sure, but effectively _check_ has not definition now, so that it does not do anything. Really this names should be aligned.